2017-2018/GameEngine/graphics/spritesheet.py

```diff
@@ -0,0 +1,37 @@
+# This class handles sprite sheets
+# This was taken from www.scriptefun.com/transcript-2-using
+# sprite-sheets-and-drawing-the-background
+# I've added some code to fail if the file wasn't found..
+# Note: When calling images_at the rect is the format:
+# (x, y, x + offset, y + offset)
+
+import pygame
+
+class spritesheet(object):
```

Comment:
https://www.python.org/dev/peps/pep-0008/#class-names

Reply:
This file was taken off the interwebz